Src/libCZI/subblock_cache.h

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: 2023 Carl Zeiss Microscopy GmbH
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#pragma once
+
+#include "libCZI.h"
+#include <map>
+#include <cstdint>
+#include <mutex>
+#include <atomic>
+
+class SubBlockCache : public libCZI::ISubBlockCache
+{
+private:
+    struct CacheEntry
+    {
+        std::shared_ptr<libCZI::IBitmapData> bitmap;
+        std::uint64_t lru_value;
```

Comment:
It may make more syntactic sense to the reader to have this value be `read_counter` and have the fact that we are using this value for a least recent update evaluation stay in `PruneByMemoryUsageAndElementCount`. This also means the value retains meaning on its own if a different pruning methodology is used.

Reply:
I added comments trying to describe the meaning and purpose of this field.